test/alphabet/composition/union_composition_detail_test.cpp

```diff
@@ -41,8 +43,56 @@
 
 using namespace seqan3;
 
+template <typename ...alphabet_types>
+class detail_union_composition : public union_composition<alphabet_types...>
+{
+public:
+    using union_composition<alphabet_types...>::alphabet_sum_size_v;
+    using union_composition<alphabet_types...>::alphabet_max_size_v;
+    using union_composition<alphabet_types...>::alphabet_prefix_sum_sizes;
+    using union_composition<alphabet_types...>::value_to_char_table_I;
+    using union_composition<alphabet_types...>::value_to_char_table;
+    using union_composition<alphabet_types...>::char_to_value_table;
+};
+
+TEST(union_composition_detail_test, alphabet_sum_size_v)
```

Comment:
I suggest to give short one-line comments for each test, where you state what the test examines.

Reply:
@rrahn Is that really necessary? We don't do that right now.